adam/visualization/positioning.py

```diff
@@ -80,13 +92,33 @@ class AxisAlignedBoundingBox:
     corners at (-1, -1, -1) and (1, 1, 1), giving the box a volume of 2(^3)
     """
 
-    center: torch.Tensor = attrib()  # tensor shape: (3,)
+    center: Parameter = attrib()  # tensor shape: (3,)
```

Comment:
a `Parameter` is a `torch.Tensor` - why change it to be more restricted? Couldn't you conceivable have a static bounding box you are not optimizing over?

Reply:
This was one of those mypy-motivated changes. It complains about calling the model's register_parameter function with a Tensor instead of a Parameter

Reply:
Okay. The really correct way to handle this would be to make `AxisAlignedBoundingBox` a generic class with the center point type as a type parameter upper-bounded by `Tensor`.  But you can leave it as is until we need it to be more flexible.